project.yml

```
@@ -207,48 +184,13 @@ targets:
     dependencies:
       - sdk: SwiftUI.framework
       - sdk: WidgetKit.framework
-      - framework: Carthage/Build/iOS/abseil.framework
-        embed: false
-      - framework: Carthage/Build/iOS/BoringSSL-GRPC.framework
-        embed: false
-      - framework: Carthage/Build/iOS/FIRAnalyticsConnector.framework
-        embed: false
-      - framework: Carthage/Build/iOS/FirebaseAnalytics.framework
-        embed: false
-      - framework: Carthage/Build/iOS/FirebaseCore.framework
-        embed: false
-      - framework: Carthage/Build/iOS/FirebaseCoreDiagnostics.framework
-        embed: false
-      - framework: Carthage/Build/iOS/FirebaseCrashlytics.framework
-        embed: false
-      - framework: Carthage/Build/iOS/FirebaseFirestore.framework
-        embed: false
-      - framework: Carthage/Build/iOS/FirebaseInstallations.framework
-        embed: false
-      - framework: Carthage/Build/iOS/FirebaseRemoteConfig.framework
-        embed: false
-      - framework: Carthage/Build/iOS/GoogleAppMeasurement.framework
-        embed: false
-      - framework: Carthage/Build/iOS/GoogleDataTransport.framework
-        embed: false
-      - framework: Carthage/Build/iOS/GoogleToolboxForMac.framework
-        embed: false
-      - framework: Carthage/Build/iOS/GoogleUtilities.framework
-        embed: false
-      - framework: Carthage/Build/iOS/gRPC-C++.framework
-        embed: false
-      - framework: Carthage/Build/iOS/gRPC-Core.framework
-        embed: false
-      - framework: Carthage/Build/iOS/GTMSessionFetcher.framework
-        embed: false
-      - framework: Carthage/Build/iOS/leveldb-library.framework
-        embed: false
-      - framework: Carthage/Build/iOS/nanopb.framework
-        embed: false
-      - framework: Carthage/Build/iOS/PromisesObjC.framework
-        embed: false
-      - framework: Carthage/Build/iOS/Protobuf.framework
-        embed: false
+      - package: FirebaseAnalytics
+      - package: FirebaseAppDistribution-Beta
+      - package: FirebaseCrashlytics
+      - package: FirebaseFirestore
+      - package: FirebaseInstallations
+      - package: FirebaseMessaging
+      - package: FirebaseRemoteConfig
```

```suggestion
```suggestion
      - package: Firebase
        product: FirebaseAnalytics
      - package: Firebase
        product: FirebaseAppDistribution-Beta
      - package: Firebase
        product: FirebaseCrashlytics
      - package: Firebase
        product: FirebaseFirestore
      - package: Firebase
        product: FirebaseInstallations
      - package: Firebase
        product: FirebaseMessaging
      - package: Firebase
        product: FirebaseRemoteConfig
```
```